kf_lib_data_ingest/etl/stage_analyses.py

```diff
@@ -70,15 +65,57 @@ def check_counts(discovery_sources, expected_counts):
                 },
                 ignore_index=True
             )
-        message.append(
+        logger.info(
+            'EXPECTED COUNT CHECKS\n' +
             tabulate(
                 checks,
                 headers='keys', showindex=False, tablefmt='psql'
             )
         )
 
-    return passed, '\n'.join(message)
+    return passed
 
 
-def compare_counts(discovery_sources_one, discovery_sources_two):
-    False, "Compare Counts NYI"
+def compare_counts(
+    name_one, discovery_sources_one, name_two, discovery_sources_two, logger
+):
+    logger.info(f'COMPARING COUNTS')
+    passed = True
+
+    setA = set(discovery_sources_one.keys())
+    setB = set(discovery_sources_two.keys())
+    diff = setA ^ setB
+    if diff:
+        logger.info(
+            f'❌ Column keys not equal between {name_one} and '
```

Comment:
So this is weird...in my text editor and log its showing there is no space between ❌and Column, however in this diff preview there is actually a space. 

Here's what my log looks like
![image](https://user-images.githubusercontent.com/10283712/56686992-4849a880-66a3-11e9-8157-8cf38f64a81a.png)

Reply:
Also can `Column keys` -> `Column names`?